OpenRA.Mods.Common/Traits/Render/WithSupportPowerActivationAnimation.cs

```diff
@@ -14,8 +14,8 @@
 
 namespace OpenRA.Mods.Common.Traits.Render
 {
-	[Desc("Replaces the building animation when `NukePower` is triggered.")]
-	public class WithNukeLaunchAnimationInfo : ConditionalTraitInfo, Requires<WithSpriteBodyInfo>
+	[Desc("Replaces the building animation when a `SupportPower` is triggered.")]
```

Comment:
I don't think Support Power needs to be in `s here. It is no longer a literal trait name.

Reply:
Renamed.